assignment7.txt

```diff
@@ -0,0 +1,16 @@
+1)
+    ++*p increments the value that p is pointing to
+    *p++ gets the value that p is pointing to, then increments p
+    *++p increments p, then returns the new value that p is pointing at
+2) yes?
```

Comment:
The simple answer to this question is neither. C doesn’t always evaluate left-to-right or right-to-left. Generally, function calls are evaluated first, followed by complex expressions and then simple expressions.